### ISSUE TITLE
"Commit all files" does not stage unstaged files anymore

### ISSUE THREAD

**Issue by joepie91** [zaggino/brackets-git#995]
When using the "commit all files" shortcut, unstaged files appear to no longer be staged automatically - in the commit window, no diffs are displayed for the unstaged files at all.

I presume this might be a consequence of #917 .

**Comment by zaggino** [comment]
@nmorenor can you maybe take a look? this looks related to https://github.com/zaggino/brackets-git/pull/959

**Comment by nmorenor** [comment]
Sure I will take a look

**Comment by nmorenor** [comment]
@joepie91 Hi I was trying to reproduce what you see when trying the "commit all files" shortcut, but I was not able to reproduce?
![screen shot 2015-04-14 at 1 18 30 pm](https://cloud.githubusercontent.com/assets/11303670/7144137/0ffbd908-e2a9-11e4-92d8-f60b1e8ff611.png)

Could you please provide more details on what you see on your side, as I understand from your previous message you do not see the changes on the commit dialog, is that correct ? I tried the following cases and it worked for me.

1. Change a file & save it.
2. use "Commit all files" shortcut

1. Change a file & save it.
2. Stage the file by selecting the combo box inside the brackets git Panel
3. Change a different file & save it
4. use the "Commit all files" shortcut

In these two cases I see the dialog as posted above, perhaps there is another case I'm not trying.

**Comment by nmorenor** [comment]
Could it be that the files you do not see are new files on the repository?

**Comment by joepie91** [comment]
@nmorenor Yes, this is specifically about new files that hadn't been committed before. Not sure why I didn't mention that, sorry for that!

Previously, those files would be added automatically. Now, they are not staged and do not appear in the diff either.
